third_party/cyw30739_sdk/merge_hex.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+#
+# Copyright 2016-2022, Cypress Semiconductor Corporation (an Infineon company) or
```

Comment:
if this is in CHIP code, should contain CHIP license attribution.

Reply:
done.